tests/runtime/ordering.test.ts

```diff
@@ -33,7 +33,7 @@ it('with whitelist', async () => {
 
   const Query = queryType({
     definition(t) {
-      // @ts-ignore
+      // @ts-expect-error
```

Comment:
If for some reason `c: true` doesn't raise an error then TS will raise an error because it didn't because of our annotation here. This is an improvement from before where `@ts-ignore` won't complain if its not actually doing anything.